source/types/Encryption.ts

```diff
@@ -1,4 +1,8 @@
-export type EncryptionType = "decrypted" | "privateAesKey" | "symmetricKey";
+export enum EncryptionType {
+  Decrypted = "decrypted",
+  PrivateAesKey = "privateAesKey",
+  SymmetricKey = "symmetricKey",
```

Comment:
Is this like constants. Then maybe this could be all capitalized?

Reply:
Yep they work like constants so all-caps is a good naming convention here. I assume with underscores then; `PRIVATE_AES_KEY`, `SYMMETRIC_KEY` etc? The same can be done for the case statuses.

Reply:
Brilliant!

Reply:
> Typescript themselves write their examples like we do here. But I think that we can decide on our own how we want to define enums.
> 
> https://www.typescriptlang.org/docs/handbook/enums.html#numeric-enums

Good note! Though I prefer SCREAMING_SNAKE_CASE as well. It's more in line with how constants are styled in most other languages/style guides, and it makes it clear that it is a constant value.